lib/LANraragi.pm

```diff
@@ -179,7 +184,18 @@ sub startup {
     # (https://stackoverflow.com/questions/60814220/how-to-manage-myself-sigint-and-sigterm-signals)
     $self->hook(
         before_dispatch => sub {
+            my $c = shift;
             state $unused = add_sigint_handler();
+
+            my $prefix = $self->LRR_CONF->get_baseurl;
```

Comment:
Small optimization here but ig you could use LRR_BASEURL directly?

Reply:
You're right. Probably not a material difference in processing time, but this does run every request, so it's good form.

Reply:
Well, now that we went with the cookie approach I wonder if the extra helper is necessary at all now..  
But I guess it doesn't really hurt keeping it in.